docker/.env.example

```
@@ -0,0 +1,31 @@
+
+# === Runtime ===
+TZ=Europe/Paris
+DEBUG=false
+
+# === Poller identity ===
+NAME=my-poller
+GORGONE_UID=       # Poller UID from Central (required)
+GORGONE_TOKEN=     # Gorgone token from Central (required)
+
+# === Central connection ===
+CENTRAL_HOST=      # IP or hostname of Central (required)
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)